CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+
+For some fairly complicated legal reasons we actually have three CLAs.  Which
+you sign depends on whose behalf you are contributing:
+
+- All individuals (except employees of the U.S. government – see below) should
+  sign the [Individual Contributor License Agreement](http://www.projectcalico.org/community/Individual-Contributor-Agreement).
+- If you are contributing on behalf of a company or organization, *you still
+  need to sign the Individual Contributor License Agreement* as above, but
+  someone at your company or organization also needs to sign the
+  [Corporate Contributor License Agreement](http://www.projectcalico.org/community/corporate-contributor-agreement/)
+  providing a list of people authorized to commit code to Project Calico.
+- Employees of the U.S. government do not sign the Individual Contributor
+  License Agreement.  Instead, someone with authority to sign on behalf of your
+  agency should sign the [U.S Government Contributor License Agreement](http://www.projectcalico.org/community/us-government-contributor-agreement).
+
+It is important to re-iterate: until you sign the required agreements we cannot
+accept your contribution!
+
+## Mailing lists and chat
+
+A great way to talk to us to ask questions, discuss features and bounce ideas
+is to join one of the channels listed below:
+
+* [Technical Mailing List](http://lists.projectcalico.org/mailman/listinfo/calico-tech_lists.projectcalico.org)
+* [Slack Calico Users Channel](https://calicousers.slack.com) ([Sign up](https://calicousers-slackin.herokuapp.com))
+* IRC - [#calico](https://kiwiirc.com/client/irc.freenode.net/#calico)
+
+## Reporting issues
+
+Before raising an issue with *Calico for containers*, please check for 
+duplicate issues, and read our [Troubleshooting](Troubleshooting.md)
+and our [Frequently Asked Questions](FAQ.md) documents.
+
+If you have a question, please hop on to our IRC or Slack channels.
+
+If you do need to raise an issue, please include any of the following that may
+be relevant to your problem (including too much information is never
+a bad thing):
+
+-  A detailed description of the problem.
+-  The steps to reproduce the problem (e.g. the full list of `calicoctl`
+   commands that were run)
+-  The expected result and actual result.
+-  Versions of appropriate binaries and libraries.  For example, the output from
+   `calicoctl version`, your version of Docker, rkt, Kubernetes etc.
+-  A link to any diagnostics (e.g. if using `calicoctl`, you can gathered
+   diagnostics using `calicoctl diags` - this provides instructions for 
+   uploading the diags bundle to transfer.sh - or alternatively if the 
+   diagnostics contains sensitive information we can set up an alternative
+   method for transfer
+-  If using `calicoctl` the output from `calicoctl status` run on each node
+   might also be useful.
+-  Details of your OS
+-  Environment details such as GCE, bare metal, VirtualBox.
+   
+
+## Contributing: Code and Documentation
```

Comment:
i personally prefer having the contact us stuff close to the top - we should be encouraging people to disucss with us before raising issues or writing code so it's good to let people know there are these channels.

I think this title should remove the ":" though, so it becomes Contributing Code and Documentation